protobuf/grpc_service.proto

```diff
@@ -1013,6 +1013,45 @@ message InferStatistics
   StatisticDuration cache_miss = 8;
 }
 
+//@@
+//@@.. cpp:var:: message InferResponseStatistics
+//@@
+//@@   Statistics per decoupled response.
```

Comment:
I'd also remove mentions of `decoupled` in this file too

Reply:
Updated.